depends/packages/native_cxxbridge.mk

```diff
@@ -0,0 +1,17 @@
+package=native_cxxbridge
+$(package)_version=1.0.68
+$(package)_download_path=https://github.com/dtolnay/cxx/archive/refs/tags
+$(package)_file_name=native_cxxbridge-$($(package)_version).tar.gz
+$(package)_download_file=$($(package)_version).tar.gz
+$(package)_sha256_hash=6fed9ef1c64a37c343727368b38c27fa4e15b27ca9924c6672a6a5496080c832
+$(package)_build_subdir=gen/cmd
+$(package)_dependencies=native_rust
+
+define $(package)_build_cmds
+  cargo build --release
+endef
+
+define $(package)_stage_cmds
+  mkdir -p "$($(package)_staging_prefix_dir)" && \
+  cargo install --path=. --root=$($(package)_staging_prefix_dir)
+endef
```

Comment:
The build and install commands here can be made `--offline` right?

Reply:
I actually have a local TODO to vendor the dependencies of this. So no, we can't use offline mode for this yet.

Reply:
This has now been done.